flake.nix

```
@@ -261,6 +261,11 @@
         };
     } // utils.lib.eachDefaultSystem (system:
       let
+	rocksdbOverlay = pkgs: prev:
+          if builtins.elem system [ "aarch64-darwin" "x86_64-darwin" ] then
```

This change is an optional suggestion
```suggestion
          if prev.stdenv.isDarwin then
```